ML-Transformers/BackgroundEnchantments/main.ts

```diff
@@ -162,7 +162,7 @@ async function main() {
   githubButton.addEventListener("click", () => {
     window
       .open(
-        "https://github.com/Vonage/vonage-media-transformers-samples/tree/main/examples/mediapipe/BackgroundEnchantments",
+        "https://github.com/Vonage/vonage-media-transformers-samples/tree/main/ML-Transformers/BackgroundEnchantments",
```

Comment:
This change was made to all the index.hash.js files below with the large diff too, the diff won't show it since the whole bundle is on one line though.